Magento_Contact/templates/form.phtml

```diff
@@ -0,0 +1,89 @@
+<?php
+// @codingStandardsIgnoreFile
+
+?>
+<div class="container">
+    <form class="form contact"
+          action="<?php /* @escapeNotVerified */ echo $block->getFormAction(); ?>"
+          id="contact-form"
+          method="post"
+          data-hasrequired="<?php /* @escapeNotVerified */ echo __('* Required Fields') ?>"
+          data-mage-init='{"validation":{}}'
+    >
+        <div class="heading heading--fourth-level">
+            <?php /* @escapeNotVerified */ echo __('Write Us') ?>
+        </div>
```

Comment:
maybe we will use some heading here?, we have `h1` already as a page title, I think we even should use `h2` here, it can be styled as  `h4`. with  headings tag, the website will e more accessible